### ISSUE TITLE
Codepoint 984245 not found in font, aborting.

### ISSUE THREAD

**Issue by yang-lile** [flutter/flutter#98836]
## introduction

I try to show the all of Icons elements, but has something bug.

1. I use https://github.com/yang-lile/flutter_assistant/blob/main/cli_tools/icons_gen.dart to generate icons.
2. show it by a IconList class at https://github.com/yang-lile/flutter_assistant/blob/main/lib/home_page.dart#L140

code in my repository: https://github.com/yang-lile/flutter_assistant

## Steps to Reproduce

1. when run my code. It running.

> flutter run
```
Flutter assets will be downloaded from https://storage.flutter-io.cn. Make sure you trust this source!
Launching lib\main.dart on M2012K11AC in debug mode...
Flutter assets will be downloaded from https://storage.flutter-io.cn. Make sure you trust this source!
Running Gradle task 'assembleDebug'...                             35.4s
✓  Built build\app\outputs\flutter-apk\app-debug.apk.
Syncing files to device M2012K11AC...                              191ms

Flutter run key commands.
r Hot reload. 🔥🔥🔥
R Hot restart.
h List all available interactive commands.
d Detach (terminate "flutter run" but leave application running).
c Clear the screen
q Quit (terminate the application on the device).

💪 Running with sound null safety 💪

An Observatory debugger and profiler on M2012K11AC is available at: http://127.0.0.1:60692/Zud75tABAFU=/
The Flutter DevTools debugger and profiler on M2012K11AC is available at:
http://127.0.0.1:9101?uri=http://127.0.0.1:60692/Zud75tABAFU=/
D/DecorView[](  923): getWindowModeFromSystem  windowmode is 1



Flutter run key commands.
r Hot reload. 🔥🔥🔥
R Hot restart.
v Open Flutter DevTools.
s Save a screenshot to flutter.png.
w Dump widget hierarchy to the console.                                               (debugDumpApp)
t Dump rendering tree to the console.                                          (debugDumpRenderTree)
L Dump layer tree to the console.                                               (debugDumpLayerTree)
S Dump accessibility tree in traversal order.                                   (debugDumpSemantics)
U Dump accessibility tree in inverse hit test order.                            (debugDumpSemantics)
i Toggle widget inspector.                                  (WidgetsApp.showWidgetInspectorOverride)
p Toggle the display of construction lines.                                  (debugPaintSizeEnabled)
I Toggle oversized image inversion.                                     (debugInvertOversizedImages)
o Simulate different operating systems.                                      (defaultTargetPlatform)
b Toggle platform brightness (dark and light mode).                        (debugBrightnessOverride)
P Toggle performance overlay.                                    (WidgetsApp.showPerformanceOverlay)
a Toggle timeline events for all widget build methods.                    (debugProfileWidgetBuilds)
M Write SkSL shaders to a unique file in the project directory.
g Run source code generators.
h Repeat this help message.
d Detach (terminate "flutter run" but leave application running).
c Clear the screen
q Quit (terminate the application on the device).

💪 Running with sound null safety 💪

An Observatory debugger and profiler on M2012K11AC is available at: http://127.0.0.1:60692/Zud75tABAFU=/
The Flutter DevTools debugger and profiler on M2012K11AC is available at:
http://127.0.0.1:9101?uri=http://127.0.0.1:60692/Zud75tABAFU=/

Taking screenshot for M2012K11AC...                              1,162ms
Screenshot written to flutter_01.png (110kB).

Application finished.
```
![flutter_01](https://user-images.githubusercontent.com/35729940/154859115-1a9b9250-1953-409d-bf13-121bbb19397f.png)
but some icons display exception. for example: Icons.onetwothree

2. when I build it, failed.

> flutter build apk
```
Flutter assets will be downloaded from https://storage.flutter-io.cn. Make sure you trust this source!

💪 Building with sound null safety 💪

Flutter assets will be downloaded from https://storage.flutter-io.cn. Make sure you trust this source!
Codepoint 984245 not found in font, aborting.



FAILURE: Build failed with an exception.

* Where:
Script 'D:\workspace\local\flutter\flutter\packages\flutter_tools\gradle\flutter.gradle' line: 1102

* What went wrong:
Execution failed for task ':app:compileFlutterBuildRelease'.
> Process 'command 'D:\workspace\local\flutter\flutter\bin\flutter.bat'' finished with non-zero exit value 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 7s
Running Gradle task 'assembleRelease'...                            8.2s
Gradle task assembleRelease failed with exit code 1
```
> flutter build apk -v
```
Flutter assets will be downloaded from https://storage.flutter-io.cn. Make sure you trust this source!

💪 Building with sound null safety 💪

Flutter assets will be downloaded from https://storage.flutter-io.cn. Make sure you trust this source!
Codepoint 984245 not found in font, aborting.



FAILURE: Build failed with an exception.

* Where:
Script 'D:\workspace\local\flutter\flutter\packages\flutter_tools\gradle\flutter.gradle' line: 1102

* What went wrong:
Execution failed for task ':app:compileFlutterBuildRelease'.
> Process 'command 'D:\workspace\local\flutter\flutter\bin\flutter.bat'' finished with non-zero exit value 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 7s
Running Gradle task 'assembleRelease'...                            8.2s
Gradle task assembleRelease failed with exit code 1
PS D:\workspace\ProjuctFiles\flutter_assistant> flutter build apk -v
[  +86 ms] executing: [D:\workspace\local\flutter\flutter/] git -c log.showSignature=false log -n 1 --pretty=format:%H
[ +102 ms] Exit code 0 from: git -c log.showSignature=false log -n 1 --pretty=format:%H
[   +1 ms] db747aa1331bd95bc9b3874c842261ca2d302cd5
[   +1 ms] executing: [D:\workspace\local\flutter\flutter/] git tag --points-at db747aa1331bd95bc9b3874c842261ca2d302cd5
[  +83 ms] Exit code 0 from: git tag --points-at db747aa1331bd95bc9b3874c842261ca2d302cd5
[        ] 2.10.1
[  +12 ms] executing: [D:\workspace\local\flutter\flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +62 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] origin/stable
[        ] executing: [D:\workspace\local\flutter\flutter/] git ls-remote --get-url origin
[  +51 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] https://github.com/flutter/flutter.git
[ +157 ms] executing: [D:\workspace\local\flutter\flutter/] git rev-parse --abbrev-ref HEAD
[  +67 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] stable
[ +114 ms] Artifact Instance of 'AndroidGenSnapshotArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[   +8 ms] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[   +9 ms] Flutter assets will be downloaded from https://storage.flutter-io.cn. Make sure you trust this source!
[  +78 ms] Artifact Instance of 'MaterialFonts' is not required, skipping update.
[        ] Artifact Instance of 'GradleWrapper' is not required, skipping update.
[   +9 ms] Artifact Instance of 'AndroidInternalBuildArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterWebSdk' is not required, skipping update.
[        ] Artifact Instance of 'FlutterSdk' is not required, skipping update.
[        ] Artifact Instance of 'WindowsEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'WindowsUwpEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxEngineArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'LinuxFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'MacOSFuchsiaSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerSDKArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FlutterRunnerDebugSymbols' is not required, skipping update.
[        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'IosUsbArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'FontSubsetArtifacts' is not required, skipping update.
[        ] Artifact Instance of 'PubDependencies' is not required, skipping update.
[ +144 ms] Skipping pub get: version match.
[  +47 ms] Found plugin path_provider at
C:\Users\lilua\AppData\Roaming\Pub\Cache\hosted\pub.flutter-io.cn\path_provider-2.0.9\
[   +7 ms] Found plugin path_provider_android at
C:\Users\lilua\AppData\Roaming\Pub\Cache\hosted\pub.flutter-io.cn\path_provider_android-2.0.11\
[   +3 ms] Found plugin path_provider_ios at
C:\Users\lilua\AppData\Roaming\Pub\Cache\hosted\pub.flutter-io.cn\path_provider_ios-2.0.7\
[   +3 ms] Found plugin path_provider_linux at
C:\Users\lilua\AppData\Roaming\Pub\Cache\hosted\pub.flutter-io.cn\path_provider_linux-2.1.5\
[   +3 ms] Found plugin path_provider_macos at
C:\Users\lilua\AppData\Roaming\Pub\Cache\hosted\pub.flutter-io.cn\path_provider_macos-2.0.5\
[   +5 ms] Found plugin path_provider_windows at
C:\Users\lilua\AppData\Roaming\Pub\Cache\hosted\pub.flutter-io.cn\path_provider_windows-2.0.5\
[  +77 ms] Found plugin path_provider at
C:\Users\lilua\AppData\Roaming\Pub\Cache\hosted\pub.flutter-io.cn\path_provider-2.0.9\
[   +3 ms] Found plugin path_provider_android at
C:\Users\lilua\AppData\Roaming\Pub\Cache\hosted\pub.flutter-io.cn\path_provider_android-2.0.11\
[   +2 ms] Found plugin path_provider_ios at
C:\Users\lilua\AppData\Roaming\Pub\Cache\hosted\pub.flutter-io.cn\path_provider_ios-2.0.7\
[   +2 ms] Found plugin path_provider_linux at
C:\Users\lilua\AppData\Roaming\Pub\Cache\hosted\pub.flutter-io.cn\path_provider_linux-2.1.5\
[   +1 ms] Found plugin path_provider_macos at
C:\Users\lilua\AppData\Roaming\Pub\Cache\hosted\pub.flutter-io.cn\path_provider_macos-2.0.5\
[   +4 ms] Found plugin path_provider_windows at
C:\Users\lilua\AppData\Roaming\Pub\Cache\hosted\pub.flutter-io.cn\path_provider_windows-2.0.5\
[  +28 ms] Generating
D:\workspace\ProjuctFiles\flutter_assistant\android\app\src\main\java\io\flutter\plugins\GeneratedPluginRegistrant.java
[  +97 ms] 💪 Building with sound null safety 💪
[  +28 ms] Running Gradle task 'assembleRelease'...
[   +6 ms] Using gradle from D:\workspace\ProjuctFiles\flutter_assistant\android\gradlew.bat.
[  +18 ms] executing: D:\App\android studio\jre\bin\java -version
[ +166 ms] Exit code 0 from: D:\App\android studio\jre\bin\java -version
[        ] openjdk version "11.0.11" 2021-04-20
           OpenJDK Runtime Environment (build 11.0.11+9-b60-7590822)
           OpenJDK 64-Bit Server VM (build 11.0.11+9-b60-7590822, mixed mode)
[   +2 ms] executing: D:\App\android studio\jre\bin\java -version
[ +150 ms] Exit code 0 from: D:\App\android studio\jre\bin\java -version
[        ] openjdk version "11.0.11" 2021-04-20
           OpenJDK Runtime Environment (build 11.0.11+9-b60-7590822)
           OpenJDK 64-Bit Server VM (build 11.0.11+9-b60-7590822, mixed mode)
[   +3 ms] executing: [D:\workspace\ProjuctFiles\flutter_assistant\android/]
D:\workspace\ProjuctFiles\flutter_assistant\android\gradlew.bat -Pverbose=true
-Ptarget-platform=android-arm,android-arm64,android-x64 -Ptarget=lib\main.dart
-Pbase-application-name=android.app.Application -Pdart-obfuscation=false -Ptrack-widget-creation=true
-Ptree-shake-icons=true assembleRelease
[+4485 ms] > Task :app:compileFlutterBuildRelease
[        ] Flutter assets will be downloaded from https://storage.flutter-io.cn. Make sure you trust this source!
[+2379 ms] Codepoint 984245 not found in font, aborting.
[ +396 ms] > Task :app:compileFlutterBuildRelease FAILED
[   +1 ms] FAILURE: Build failed with an exception.
[        ] * Where:
[        ] Script 'D:\workspace\local\flutter\flutter\packages\flutter_tools\gradle\flutter.gradle' line: 1102
[        ] * What went wrong:
[        ] Execution failed for task ':app:compileFlutterBuildRelease'.
[   +1 ms] Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.
[        ] You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from
your own scripts or plugins.
[   +1 ms] See https://docs.gradle.org/7.4/userguide/command_line_interface.html#sec:command_line_warnings
[        ] 1 actionable task: 1 executed
[   +1 ms] > Process 'command 'D:\workspace\local\flutter\flutter\bin\flutter.bat'' finished with non-zero exit value 1
[        ] * Try:
[        ] > Run with --stacktrace option to get the stack trace.
[        ] > Run with --info or --debug option to get more log output.
[        ] > Run with --scan to get full insights.
[        ] * Get more help at https://help.gradle.org
[        ] BUILD FAILED in 7s
[ +561 ms] Running Gradle task 'assembleRelease'... (completed in 8.2s)
[   +5 ms] "flutter apk" took 8,817ms.
[   +6 ms] Gradle task assembleRelease failed with exit code 1
[        ]
           #0      throwToolExit (package:flutter_tools/src/base/common.dart:10:3)
           #1      AndroidGradleBuilder.buildGradleApp (package:flutter_tools/src/android/gradle.dart:400:9)
           <asynchronous suspension>
           #2      AndroidGradleBuilder.buildApk (package:flutter_tools/src/android/gradle.dart:179:5)
           <asynchronous suspension>
           #3      BuildApkCommand.runCommand (package:flutter_tools/src/commands/build_apk.dart:112:5)
           <asynchronous suspension>
           #4      FlutterCommand.run.<anonymous closure>
(package:flutter_tools/src/runner/flutter_command.dart:1161:27)
           <asynchronous suspension>
           #5      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
           <asynchronous suspension>
           #6      CommandRunner.runCommand (package:args/command_runner.dart:209:13)
           <asynchronous suspension>
           #7      FlutterCommandRunner.runCommand.<anonymous closure>
           (package:flutter_tools/src/runner/flutter_command_runner.dart:281:9)
           <asynchronous suspension>
           #8      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
           <asynchronous suspension>
           #9      FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:229:5)
           <asynchronous suspension>
           #10     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:62:9)
           <asynchronous suspension>
           #11     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:150:19)
           <asynchronous suspension>
           #12     main (package:flutter_tools/executable.dart:94:3)
           <asynchronous suspension>


[ +119 ms] ensureAnalyticsSent: 116ms
[   +3 ms] Running shutdown hooks
[        ] Shutdown hooks complete
[        ] exiting with code 1
```

**Comment by danagbemava-nc** [comment]
Hi @yang-lile, please provide the output of `flutter doctor -v`. 

Also, can you provide what exactly the steps to reproduce are? Do I just build the app or do I have to generate the icons first?

**Comment by yang-lile** [comment]
@danagbemava-nc thanks.
> flutter doctor -v
```
╔════════════════════════════════════════════════════════════════════════════╗
║ A new version of Flutter is available!                                     ║
║                                                                            ║
║ To update to the latest version, run "flutter upgrade".                    ║
╚════════════════════════════════════════════════════════════════════════════╝


Flutter assets will be downloaded from https://storage.flutter-io.cn. Make sure you trust this source!
[✓] Flutter (Channel stable, 2.10.1, on Microsoft Windows [Version 10.0.22000.282], locale zh-CN)
    • Flutter version 2.10.1 at D:\workspace\local\flutter\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision db747aa133 (12 days ago), 2022-02-09 13:57:35 -0600
    • Engine revision ab46186b24
    • Dart version 2.16.1
    • DevTools version 2.9.2
    • Pub download mirror https://pub.flutter-io.cn
    • Flutter download mirror https://storage.flutter-io.cn

[✓] Android toolchain - develop for Android devices (Android SDK version 32.0.0)
    • Android SDK at D:\workspace\local\Android\android-sdk-windows
    • Platform android-32, build-tools 32.0.0
    • ANDROID_HOME = D:\workspace\local\Android\android-sdk-windows
    • Java binary at: D:\App\android studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.11+9-b60-7590822)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[✓] Visual Studio - develop for Windows (Visual Studio Community 2019 16.11.6)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Community
    • Visual Studio Community 2019 version 16.11.31829.152
    • Windows 10 SDK version 10.0.19041.0

[✓] Android Studio (version 2021.1)
    • Android Studio at D:\App\android studio
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.11+9-b60-7590822)

[✓] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.22000.282]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 98.0.4758.102
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 98.0.1108.56

[!] HTTP Host Availability
    ✗ HTTP host https://maven.google.com/ is not reachable. Reason: An error occurred while checking the HTTP host:
      信号灯超时时间已到


! Doctor found issues in 1 category.
```

**Comment by yang-lile** [comment]
the steps to reproduce:

1. clone from my repository: https://github.com/yang-lile/flutter_assistant
2. `cd flutter_assistant`
2. `dart run cli_tools/icons_gen.dart`
3. `flutter pub get`
4. `flutter run`

[assets/source/icons.dart](https://github.com/yang-lile/flutter_assistant/blob/main/assets/source/icons.dart) is copy from [icons.dart](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/icons.dart), then generate [lib/gen/icons.g.dart](https://github.com/yang-lile/flutter_assistant/blob/main/lib/gen/icons.g.dart) by [cli_tools/icons_gen.dart](https://github.com/yang-lile/flutter_assistant/blob/main/cli_tools/icons_gen.dart).

that all @danagbemava-nc, thanks!

**Comment by danagbemava-nc** [comment]
Hi @yang-lile, there was an issue with the icons in `flutter 2.10.0` which has been fixed with `flutter 2.10.2`. Can you upgrade to that version and let me know whether or not you still experience this issue?

FWIW: for me, on flutter 2.10.2 all the icons are properly rendered and the apk built just fine.

**Comment by yang-lile** [comment]
Perfect solution! thanks sir! @danagbemava-nc
